### DISCUSSION TITLE
Regarding the extensions of the xdm

### DISCUSSION THREAD

**Discussion by RiotRonzoX** [subhra74/xdm#980] [Q&A]
I am trying to add the extension using the browser monitoring link on chrome. But it always gives me 404 error. Please help.

**Comment by johngerome** [comment]
Same here

**Comment by lezzuhls** [comment]
Same here. Chromium, Chrome, and Opera links are bad. Also tried searching "xtreme-download-manager" (title in the Chrome Extension link) in the Chrome store but no results for this app.

https://subhra74.github.io/xdm/redirect.html?target=chrome (link for Chrome)
https://subhra74.github.io/xdm-firefox/chromium.html (link for Chromium and Opera)

**Comment by ColtonYYZ** [comment]
Because you have to go here to install the beta version:  https://github.com/subhra74/xdm-experimental-binaries/releases

**Reply by wangjianfei130** [reply]
no MAC version

**Comment by JuanCarlosM1** [comment]
If you are using the old version you cloud use the Download From Edge Store.
https://microsoftedge.microsoft.com/addons/detail/xdm-browser-monitor/plohkbgcfpdjhcnoahppkcffaijkadgf?hl=pt-BR
I guess the exstension are avaliable too on the XDM folder

**Comment by Vag-Soft** [comment]
The extension had disappeared for me too.
I tried downloading it again and I faced the same issues as everybody else.

Thankfully, I found it still installed, in chrome://extensions/, but it was disabled.
I enabled it and it works once again now.
